src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -320,7 +320,7 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
-                {!isSmallScreenWidth && getHeaderText()}
+                {!isSmallScreenWidth || (tagList.length === 0 && !isLoading) && getHeaderText()}
```

Comment:
Can you do same for the categories too?

Reply:
The `isSmallScreenWidth` check is added later in the `SelectionScreen` `listHeaderComponent`. It seems it was added intentionally. So not sure about that.